regulations/tasks.py

```diff
@@ -125,22 +126,40 @@ def html_to_pdf(html):
         shutil.rmtree(path)
 
 
-def cache_pdf(pdf, metadata_url):
+def cache_pdf(pdf, document_number, metadata_url):
     """Update submission metadata and cache comment PDF."""
     url = SignedUrl.generate()
+    content_disposition = generate_content_disposition(document_number,
+                                                       draft=False)
     s3_client.put_object(
         Body=json.dumps({'pdfUrl': metadata_url.url}),
         Bucket=settings.ATTACHMENT_BUCKET,
         Key=metadata_url.key,
     )
     s3_client.put_object(
         Body=pdf,
+        ContentType='application/pdf',
+        ContentDisposition=content_disposition,
         Bucket=settings.ATTACHMENT_BUCKET,
         Key=url.key,
     )
     return url
 
 
+def generate_content_disposition(document_number, draft=False):
+    return 'attachment; filename="{}comment_{}.pdf"'.format(
+        "DRAFT_" if draft else "",
```

Comment:
Right now, it yields:

- DRAFT_comment_2016_02749.pdf
- comment_2016_02749.pdf

_DRAFT_ is in front.